ApiCalls.js

```diff
@@ -136,3 +136,25 @@ export const deleteFish = async (fishId) => {
     console.error(error);
   }
 };
+
+// Image upload for species identification:
+export const uploadPhoto = async (photo) => {
+  try {
+    const response = await fetch(
+      "https://my-fly-box-api.herokuapp.com/api/v1/images",
```

Comment:
A good call to move this to the ApiCalls.  This make it the proper formatting for the fetch calls.